package.json

```diff
@@ -33,27 +33,29 @@
   },
   "homepage": "https://github.com/cytoscape/cytoscape.js-dagre",
   "devDependencies": {
-    "babel-core": "^6.24.1",
-    "babel-loader": "^7.0.0",
-    "babel-preset-env": "^1.5.1",
-    "camelcase": "^4.1.0",
-    "chai": "4.0.2",
-    "cpy-cli": "^1.0.1",
-    "cross-env": "^5.0.0",
-    "eslint": "^3.9.1",
-    "gh-pages": "^1.0.0",
-    "mocha": "3.4.2",
-    "npm-run-all": "^4.1.2",
-    "rimraf": "^2.6.2",
+    "@webpack-cli/serve": "^0.2.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.1.0",
+    "babel-preset-env": "^1.7.0",
+    "camelcase": "^6.0.0",
+    "chai": "4.2.0",
+    "cpy-cli": "^3.1.1",
+    "cross-env": "^7.0.2",
+    "eslint": "^7.6.0",
+    "gh-pages": "^3.1.0",
+    "mocha": "8.1.1",
+    "npm-run-all": "^4.1.5",
+    "rimraf": "^3.0.2",
     "update": "^0.7.4",
     "updater-license": "^1.0.0",
-    "webpack": "^2.7.0",
-    "webpack-dev-server": "^2.4.5"
+    "webpack": "^4.44.1",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-cli": "^3.3.12"
   },
   "peerDependencies": {
-    "cytoscape": "^3.2.0"
+    "cytoscape": "^3.15.2"
   },
   "dependencies": {
-    "dagre": "^0.8.2"
+    "dagre": "^0.8.5"
```

Comment:
Updating the version of Dagre is great.  It will require a feature version update to 2.3.0 of this extension.